test/IDE/complete_macro_declaration.swift

```diff
@@ -0,0 +1,22 @@
+// REQUIRES: swift_swift_parser
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-ide-test -batch-code-completion -source-filename %s -filecheck %raw-FileCheck -completion-output-dir %t
+
+let globalVar = 1
+macro expect(file: Int = #^DEFAULT_ARG^#) = #externalMacro(module: "MyModule", type: "MyMacro")
+// DEFAULT_ARG: Decl[GlobalVar]/CurrModule/TypeRelation[Convertible]: globalVar[#Int#]; name=globalVar
+
+@freestanding(expression)
+macro externalMacro() = ##^EXTERNAL_MACRO^#externalMacro
```

Comment:
Given the test just below this one, I assume this isn't meant to have the trailing `externalMacro`?

Reply:
Yes, you’re right